app/javascript/dashboard/routes/dashboard/helpcenter/components/Header/ArticleHeader.vue

```diff
@@ -146,16 +203,32 @@ export default {
       this.$emit('close');
       this.showSortByDropdown = false;
     },
+    openLocaleDropdown() {
+      this.$emit('open');
+      this.showLocaleDropdown = true;
+    },
+    closeLocaleDropdown() {
+      this.$emit('close');
```

Comment:
Why do we need to emit open and close events?

Reply:
This was added by mistake, fixing this.